### DISCUSSION TITLE
zotero 7 - 评级（ratings)后会显示在title最前端

### DISCUSSION THREAD

**Discussion by DaxuanCCC** [MuiseDestiny/zotero-style#1072] [Q&A]
首先感谢大佬开发的这么多实用功能！

升级Zotero7之后发现多了一个小bug，具体描述如下：

1. 每次新对一篇期刊文献进行评级之后（比如点选5颗星），这个五颗星也会显示在文章title的最前端；如果是升级之前就已经添加评级的item则不会有这种现象。
2. 类似的，一些找不到DOI号的文献会在文献title前段显示一个红圈的横杠emoji。

感觉这两个情况还挺占用空间的，请问有什么解决办法吗？

（如图）
<img width="965" alt="截屏2024-08-10 下午7 46 15" src="https://github.com/user-attachments/assets/92e9352c-70c3-4441-8f61-3f0073e78fd1">

**Comment by zhu-zhiyun** [comment]
同问，这个小bug有点烦人。

**Comment by zhu-zhiyun** [comment]
正确的版本，是在选择“评级”后，在“其他”著录项内出现“rate: 4”类似的标记。Bug版本没有“rate：数字”的标记，却是自动生成一个“标签”。

**Comment by DeciduousPZW** [comment]
不知道楼主有没有解决，我现在方法是把标签列单独出来（在栏目标题右键-标签），评级的标记就不会在前面标签里出现，这样就不会多占用空间了
![image](https://github.com/user-attachments/assets/d958fc68-ac5b-4aeb-b852-d056992e3539)

**Comment by buzaiqizhong** [comment]
有差不多的问题，之前给的5星不会显示在左下角的标签区，新的文献给星星后会同时生成标签。虽然不影响使用，但是显得标签有些乱。

**Comment by MuiseDestiny** [comment]
标题列设置-取消勾选emoj

**Reply by szhangsg** [reply]
可是标题列无法编辑。其他列均可设置，只有标题列右击后不显示column setting
